.github/workflows/develop_commit_notification.yml

```diff
@@ -18,8 +18,16 @@ jobs:
           private_key: ${{ secrets.NOTIFICATIONS_TOKEN_GENERATOR_GH_APP_PRIVATE_KEY }}
       - name: Repository Dispatch
         uses: peter-evans/repository-dispatch@87c5425cae5ba8b5bc7da27674076c78588babf3
+        env:
+          VERSION: ${{ format('auto-{0}', github.sha) }}
         with:
           token: ${{ steps.generate_token.outputs.token }}
           repository: oppia/release-scripts
           event-type: develop-commit
-          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
+          client-payload: '{
+            "versions": "${{ env.VERSION }}",
+            "app_name": "oppiaserver-backup-migration",
+            "maintenance_mode": "false",
+            "sha": "${{ github.sha }}",
+            "repo_owner": "oppia"
```

Comment:
Can we make this into a GitHub variable? `${{github.repository_owner}}`

Reply:
Sure